recipes/cheat/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cheat" %}
+{% set version = "2.1.9" %}
+{% set md5 = "3bf4c63c194c493040c5a1b0f8e5c0dc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
+
+requirements:
+  build:
+    - python
```

Comment:
Can we drop the dependencies below from the build requirements or are they needed during the build for some reason?